composer.json

```diff
@@ -13,6 +13,8 @@
         "codeigniter/framework": "^3.1",
         "paragonie/random_compat": "^9.99",
         "giggsey/libphonenumber-for-php": "^8.12",
+        "league/csv": "^8.2 || ^9.5 || ^9.6 || ^9.7 || ^9.8",
```

Comment:
Could be simply written as "^8.2 || ^9.5" 

https://getcomposer.org/doc/articles/versions.md#caret-version-range-

Reply:
I'm not sure to understand.
8.2 needs php >= 5.5
9.5 needs php >= 7.0.10
9.6 needs php >= 7.2.5
9.7 needs php >= 7.3
9.8 needs php >= 7.4

So should I just use `"^8.2 || ^9.5"` and user will get the latest version his php version supports? For example:
- If he has php7.2.5 he will get up 9.6.2 then?
- If he has php7.0.10 he will get up 9.5.0 then?

Reply:
Yes, atleast that's what the documentation says.

We can keep an eye on any reported issue just in case.

Reply:
updates, thanks.